docs/HOWTO-ar.md

```diff
@@ -1,15 +1,17 @@
 *[إقرأ هذا بلغات أخرى](../README.md#translations)*
 
+<div dir="rtl" markdown="1">
+
 مرحبا بكم في *!Free-Programming-Books* نرحّب بجميع المساهمين الجدد؛ ونرحب أيضا بهؤلاء الذين يريدون تقديم أول بولّ ريكويست لهم علي جيت هبّ. إن كنت واحدا منهم، فإليك بعض المصادر التي ربما تساعدك:
 
-* [ 🇺🇸 عن البولّ ريكويست](https://help.github.com/articles/about-pull-requests/)
```

Comment:
I'm interesting in why to remove emoji flags in which resource are :worried:

Reply:
Because they are not displayed as icons, they are just text as `EG`, `SU` 
I also noticed that it is not available in any other language except the English version, and this was strange.

Reply:
I will return them

Reply:
Ir flag or text, but not remove at all. even not translate english resource titles. Think in screen readers too.

 See other howto to normalize guide.

my opinion: `flag + *lang note)*`

Reply:
> Because they are not displayed as icons, they are just text as `EG`, `SU` I also noticed that it is not available in any other language except the English version, and this was strange.

is your browser suporting SVG? GH emoji rendering test supporting and if not a text is rendered

Reply:
my browser render them like this:
![image](https://user-images.githubusercontent.com/69223584/152891578-1a93b51d-767b-466d-b6ba-b2a04f2e19a0.png)

+ why there are no flags in others `howto`
If there are, so i think my browser doesn't support them, so I'm sorry for that mistake

Reply:
reopenend.

As Eric comments in https://github.com/EbookFoundation/free-programming-books/pull/6701#issuecomment-1032737122, it's better remove flags and add a note about lang at the end.